.prettierignore

```diff
@@ -9,6 +9,9 @@ package-lock.json
 /resources/todomvc/architecture-examples/*
 !/resources/todomvc/architecture-examples/angular
 !/resources/todomvc/architecture-examples/backbone
+!/resources/todomvc/architecture-examples/lit
+/resources/todomvc/architecture-examples/lit/dist
+/resources/todomvc/architecture-examples/lit/.wireit
```

Comment:
do we need those in .eslintignore as well?

Reply:
+1, done